src/main/java/org/thoughtcrime/securesms/ProfileAdapter.java

```diff
@@ -339,7 +340,7 @@ public void changeData(@Nullable int[] memberList, @Nullable DcContact dcContact
         } else {
           introducedBy = context.getString(R.string.verified_by, dcContext.getContact(verifierId).getDisplayName());
         }
-        itemData.add(new ItemData(ITEM_INTRODUCED_BY, introducedBy, 0));
+        itemData.add(new ItemData(ITEM_INTRODUCED_BY, introducedBy, dcContact.isVerified()? R.drawable.ic_verified : 0));
       }
```

Comment:
as discussed, the condition for showing the line should be `isVerified || getVerifierId` - one can happen sense without the other.

we need a new string "Introduced" then to cover the case where isVerified==true and getVerifierId==0

Reply:
k, thanks for clarification! then this simplifies things - but documentation was wrong, i was   up to adapt it towards "getVerifierId!=0 || isVerified" at https://github.com/chatmail/core/pull/7027 ,  but that needs more tweaking then

Reply:
if https://github.com/chatmail/core/pull/7027 is correct now, the condition here still needs to be adapted in a way that isVerified() alone creates the line